fontforge/python.c

```diff
@@ -7580,14 +7579,9 @@ Py_RETURN( self );
 static PyObject *PyFFGlyph_autoTrace(PyObject *self, PyObject *UNUSED(args)) {
     SplineChar *sc = ((PyFF_Glyph *) self)->sc;
     int layer = ((PyFF_Glyph *) self)->layer;
-    char **at_args;
-
-    at_args = AutoTraceArgs(false);
-    if ( at_args==(char **) -1 ) {
-	PyErr_Format(PyExc_EnvironmentError, "Bad autotrace args" );
-return(NULL);
```

Comment:
Personal preference I mostly, but it seemed to be breaking encapsulation - outside of `python.c`, `_SCAutoTrace` and `AutoTraceArgs` are only ever called in `autotrace.c`. Plus by calling `_SCAutoTrace` directly, it misses [other checks](https://github.com/jtanx/fontforge/blob/unidir/fontforge/autotrace.c#L487) that are made in `SCAutoTrace`.

So I guess right it's a tradeoff between less informative error messages and more error checking.